### PR TITLE
More dependencies upgrade

### PR DESCRIPTION
**Pull request by chancancode**

### CODE REVIEW

File: lib/ui/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "ember-addon"
   ],
   "dependencies": {
-    "ember-cli-babel": "^7.1.2",
```
Comment (rwwagner90) [inline lib/ui/package.json]: Do we even need this `package.json`? If we omit it, would it use the deps from the root `package.json`?

Comment (chancancode) [reply lib/ui/package.json]: We need it for the `ember-addon` stuff but I switched the deps to `*` so it just uses whatever is at the root

File: package.json [hunk 2]
```diff
@@ -96,6 +96,7 @@
     "js-string-escape": "^1.0.0",
     "loader.js": "^4.7.0",
     "mkdirp": "^0.5.1",
+    "pretender": "^3.0.4",
```
Comment (rwwagner90) [inline package.json]: Do we need to switch any imports with the switch from ember-cli-pretender to pretender?

Comment (rwjblue) [reply package.json]: No, ember-cli-pretender made `import Pretender from 'pretender'` work

Comment (chancancode) [reply package.json]: We are already importing from pretender: https://github.com/emberjs/ember-inspector/blob/master/tests/acceptance/whats-new-test.js#L4

**Pull request merged**